Lib/subprocess.py

```diff
@@ -1307,15 +1307,15 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
                 try:
                     exception_name, hex_errno, err_msg = (
                             errpipe_data.split(b':', 2))
+                    err_msg = err_msg.decode(errors="surrogatepass")
                 except ValueError:
                     exception_name = b'SubprocessError'
                     hex_errno = b'0'
```

Comment:
That makes sense, I've changed it to print out the error itself instead of the repr of the bytearray representing it. I've opted not to put it in an else block because that hurts readability in my opinion.